src/components/LoadFileMenu/style.css

```diff
@@ -30,4 +30,4 @@
     width: fit-content;
     right: 300px !important;
     left: auto !important;
```

Comment:
it would be good if you could make this selector more specific so you can avoid using the important flags

Reply:
I'm not sure what these style rules are even doing. When I remove them I can't tell that anything has changed about the pop up menu.

Might delete?

Reply:
Removed.